### PR TITLE
added search component behavior without ui

### PR DESCRIPTION
**Pull request by hanbyul-here**
- added `getSearchResult(inputText, callback)` and `getAutocompleteResult(inputText, callback)` 
- I added how to used this `examples/index.html`. This examples logs the result. I wonder there is any better way to do this. (or maybe putting this in example folder is not necessary?)

### CODE REVIEW

**Comment by rfriberg** [comment]
Nice!  I don't think it's necessary to include in `examples/index.html`, but maybe add an example for testing to our new `test/examples` directory?  Also, documentation!  :)

**Comment by hanbyul-here** [comment]
I added docs to `docs/search.md`.  Any of you has a minute to take a look at it ? @rfriberg or @kkowalsky ?

File: docs/search.md [hunk 1]
```diff
@@ -41,3 +43,13 @@ These options affect the appearance and interaction behavior of the geocoder.
 `markers` | [Leaflet Marker options object](http://leafletjs.com/reference.html#marker-options) or Boolean | `true` | If `true`, search results drops Leaflet's default blue markers onto the map. You may customize this marker's appearance and behavior using Leaflet [marker options](http://leafletjs.com/reference.html#marker-options).
 `fullWidth` | Integer or Boolean | `650` | If `true`, the input box will expand to take up the full width of the map container. If an integer breakpoint is provided, the full width applies only if the map container width is below this breakpoint.
 `place` | Boolean | `false` | If `true`, selected results will make a request to the service [`/place` endpoint](https://mapzen.com/documentation/search/place/). If `false`, this is disabled. The geocoder does not handle responses to `/place`; you will need to do handle it yourself in the `results` event listener.
+
+
+### Methods
+
+These methods work without UI component attached to the map.
```
Comment (rfriberg) [inline docs/search.md]: Looks good!  Can you add a note either in this section or at the top to explain _why_ a user might want to use the geocoder without the default UI?

**Comment by kkowalsky** [comment]
I agree with @rfriberg's comment, like the header changes.

File: docs/search.md (hunk 1 above)
Comment (hanbyul-here) [inline docs/search.md]: 🆗

**Pull request merged**